src/signals/incident/services/format-incident-message/index.js

```diff
@@ -0,0 +1,21 @@
+import flattenObject from '../object-flatten';
+
+/**
+ * Either formats message through i18n intl context or simply passes back string value
+ * @param intl
+ * @param value
+ * @param incident
+ * @returns {*}
+ */
+const formatIncidentMessage = (intl, value, incident) => {
+  if (typeof value === 'object') {
```

Comment:
Same comment about checking for type applies here.

Reply:
I made the check more specific. Though it seems we should consider a proper typing system if we're going to pin down objects in JS to this degree.